packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const PillColorStyles = {
+  "#d7d7d7": css`
```

Comment:
What about other colours like blue, green, etc.?

Reply:
Those colours are okay with a white text, in this specific colour (blackLighten60) we need it to be black to pass contrast.